package.json

```diff
@@ -221,7 +221,7 @@
     "build": "cross-env TARGET_ENV=default webpack --config webpack_config/production.js",
     "build:staging": "cross-env TARGET_ENV=staging webpack --config webpack_config/production.js",
     "prebuild": "check-node-version --package",
-    "test": "jest --config=jest_config/jest.config.json --maxWorkers=2",
+    "test": "jest --config=jest_config/jest.config.json",
```

Comment:
Can we move `jest.config.json` to the root (like in signer)?

Reply:
Out of scope